accelerometer/summariseEpoch.py

```diff
@@ -218,11 +219,14 @@ def check_daylight_savings_crossover(e, startTime, endTime, summary,
         for t in localTime._utc_transition_times:
             if t>startTime:
                 transition = t
+                transition = transition - pd.DateOffset(seconds = 1)
+                transition = utc_tz.localize(transition)
```

Comment:
In addition, even "transition" is meant to describe timestamp in UTC, it's not associated with any timezone. Here, it's just using the earlier defined "utc_tz" to give "transition" a "UTC" timezone association, so that additional timezone handling can be performed next.